mesonbuild/linkers.py

```diff
@@ -166,8 +165,11 @@ def get_std_link_args(self) -> typing.List[str]:
     def get_output_args(self, target: str) -> typing.List[str]:
         return ['-of=' + target]
 
-    def get_linker_always_args(self) -> typing.List[str]:
-        if is_windows():
+    def get_buildtype_linker_args(self, buildtype):
+        return []
```

Comment:
Is this change here on purpose or a bad squash/rebase?

Reply:
looks like a bad rebase to me I'll fix that on monday.